### PR TITLE
:sparkles: Add support for Any type

### PR DESCRIPTION
**Pull request by gkumbhat**
Signed-off-by: gkumbhat <kumbhat.gaurav@gmail.com>

### Changes
- Add support for Any type

### Tests
`pytest tests`

### CODE REVIEW

**Comment by gabe-l-hart** [comment]
@gkumbhat  Looks like you need a `make fmt`

File: jtd_to_proto/jtd_to_proto.py [hunk 1]
```diff
@@ -4,14 +4,15 @@
...
+from google.protobuf import any_pb2
 from google.protobuf import descriptor as _descriptor
 from google.protobuf import descriptor_pb2
 from google.protobuf import descriptor_pool as _descriptor_pool
 from google.protobuf import struct_pb2, timestamp_pb2
-import jtd
 
 # First Party
 import alog
+import jtd
```
Comment (gabe-l-hart) [inline jtd_to_proto/jtd_to_proto.py:15]: I think we don't need this at all

File: jtd_to_proto/jtd_to_proto.py [hunk 2]
```diff
@@ -4,14 +4,15 @@
 import re
 
 # Third Party
+from google.protobuf import any_pb2
```
Comment (gabe-l-hart) [inline jtd_to_proto/jtd_to_proto.py:7]: You can mash this import into the one that imports `struct_pb2, timestamp_pb2`. In fact, you can mash `descriptor_pb2` into that line too!

**Review by gabe-l-hart — CHANGES_REQUESTED** [review]
Pickin' NITs all day!

File: jtd_to_proto/jtd_to_proto.py (hunk 1 above)
Comment (gkumbhat) [reply jtd_to_proto/jtd_to_proto.py:15]: ohh. there is some incompatibility. Isort on my machine is making this change automatically and somehow I don't see `make fmt` failing locally as well. 🤔

File: jtd_to_proto/jtd_to_proto.py (hunk 2 above)
Comment (gkumbhat) [reply jtd_to_proto/jtd_to_proto.py:7]: hmm. was finally able to fix the formatting issue, had to try it out different machine. But it looks like `isort` doesn't allow to combine `any_pb2` with others.

File: jtd_to_proto/jtd_to_proto.py [hunk 3]
```diff
@@ -4,6 +4,7 @@
 import re
```
Comment (ethankoch4) [inline jtd_to_proto/jtd_to_proto.py:4]:
```suggestion
from ethan import the_one_function_to_rule_them_all
```

Comment (gabe-l-hart) [reply jtd_to_proto/jtd_to_proto.py:4]: You sir, are a highway bandit! (but also Hi!! 👋)

File: jtd_to_proto/jtd_to_proto.py (hunk 2 above)
Comment (gabe-l-hart) [reply jtd_to_proto/jtd_to_proto.py:7]: 🤷 if `make fmt` is happy in the CI, I'm happy

**Review by gabe-l-hart — APPROVED** [review]
@ethankoch4 can get right out of town 😉

File: jtd_to_proto/jtd_to_proto.py (hunk 3 above)
Comment (ethankoch4) [reply jtd_to_proto/jtd_to_proto.py:4]: 👋

**Pull request merged**